src/main/resources/analysis/init.xml

```diff
@@ -1,11 +1,14 @@
 <module>
-  <If condition="%{data.CREATOR}!='Ceres'">
+  <If condition="%{data.CREATOR}!='Ceres' and %{data.CREATOR}!='Ceres_Pedestal'">
     <fact.utils.PreviousEventInfo
         startCellKey="StartCellData"
         outputKey="prevEvents"
         limitEvents="${PreviousEventInfo.limitEvents}"
     />
     <fact.features.UnixTimeUTC2DateTime />
   </If>
+  <If condition="%{data.CREATOR}=='Ceres_Pedestal'">
```

Comment:
is this still needed. If I am not mistaken all processors are using ZonedDateTime now and don't need a in time stamp anymore.